.github/workflows/tests.yml

```diff
@@ -18,19 +18,17 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}    
       - name: Install and set up Poetry
-        run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+        run: |
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+          echo "PATH=$HOME/.poetry/bin:$PATH" >> $GITHUB_ENV
       - name: Install dependencies
         run: |
-          source $HOME/.poetry/env
           poetry install -vvv
           poetry add -D coverage
       - name: Lint with flake8
-        run: |
-          source $HOME/.poetry/env
-          poetry run flake8
+        run: poetry run flake8
```

Comment:
Would you mind to change it as `run: |`, even if it only has one command to run, it'll be better to keep the same style everywhere.

Reply:
Sure :+1: